server/src/main.rs

```diff
@@ -1,9 +1,22 @@
+use cen::SignedReport;
+use warp::Filter;
+
 mod storage;
 
-use cen::SignedReport;
+struct CoepiReport(bytes::Bytes);
```

Comment:
```suggestion
struct CoepiReport(bytes::Bytes);
```
I think this might be better named `ReportBytes`, since it's not specific to CoEpi and adding Bytes says that it's a serialized form